### PR TITLE
Blazor Supported Platforms topic UE pass

### PR DESCRIPTION
**Pull request by guardrex**
Addresses #19286

[Internal Review Topic](https://review.docs.microsoft.com/en-us/aspnet/core/blazor/supported-platforms?view=aspnetcore-3.1&branch=pr-en-us-20006)

Recommendations on this PR:

* We have a pointless heading level here ... let's drop it. 🔪
* Alphabetize the tables.
* We call out browsers by manufacturer and name for all but Safari, so let's display that entry as "Apple Safari ..." to match.

### CODE REVIEW

**Review by danroth27 — CHANGES_REQUESTED** [review]
A couple of comments. Also, we'll need to do a couple of updates to this doc for NET 5. I'll share the details with you shortly.

File: aspnetcore/blazor/supported-platforms.md [hunk 1]
```diff
@@ -5,40 +5,43 @@ description: Learn about the supported platforms for ASP.NET Core Blazor.
...
+| Google Chrome, including Android | Current               |
+| Microsoft Internet Explorer      | Not Supported&Dagger; |
 | Microsoft Edge                   | Current               |
 | Mozilla Firefox                  | Current               |
-| Google Chrome, including Android | Current               |
-| Safari, including iOS            | Current               |
-| Microsoft Internet Explorer      | Not Supported&dagger; |
 
-&dagger;Microsoft Internet Explorer doesn't support [WebAssembly](https://webassembly.org).
+&dagger;Resource limitations prevent testing the Blazor framework APIs with all versions of the most popular browsers. Therefore, Blazor WebAssembly is generally supported for the latest versions of browsers. If a browser incompatibility is discovered for a current Blazor framework feature, open an issue at the [ASP.NET Core GitHub repository](https://github.com/dotnet/aspnetcore/issues) with complete details of the incompatibility.
```
[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/blazor/supported-platforms.md:26]: I'm not sure if you mean just that one sentence, or you don't want to state the reason for the support policy. I assume you want it cut, so I'll do that on the next commit.

If we don't have a remark on the reason for the policy in the doc, is the reasoning in that draft text :point_up: what you would want me to tell an individual dev who opens an issue asking about it?

**Review by danroth27 — APPROVED** [review]
LGTM

**Comment by danroth27** [comment]
@bradygaster Any concerns with the browser reordering on the SignalR supported platforms page?

**Comment by guardrex** [comment]
... and @bradygaster ... do you want to leave the **Version** of the column **un-centered**, or do you want me to center that column. As an example of how it would look, you can see the current Blazor doc at ...

https://docs.microsoft.com/aspnet/core/blazor/supported-platforms

If you prefer to leave it un-centered, then I'll remove the centering in the Blazor doc to match.

**Comment by bradygaster** [comment]
@guardrex centering is not required. I trust your design eye. :)

**Comment by guardrex** [comment]
Thanks @bradygaster ... what about DR's question :point_up:?

**Comment by bradygaster** [comment]
all good. [i'd have had a smart-aleck response had I disagreed with the fabulous mr. @danroth27]

**Comment by guardrex** [comment]
@danroth27 ...

> we'll need to do a couple of updates to this doc for NET 5. I'll share the details with you shortly.

I just need that bit to wrap this one up.

**Comment by danroth27** [comment]
@guardrex Blazor browser support updates for .NET 5: https://github.com/aspnet/Announcements/issues/441

**Comment by guardrex** [comment]
@danroth27 :+1: Thx ... I'll update based on that and ping u back for review tomorrow.

**Comment by guardrex** [comment]
@danroth27 Updates for .NET 5 are on the PR.

Note that I collapse the footnote lines now to one footnote per line with only a bare return among the lines (i.e., no blank line between them). It's done via two spaces at the ends of any line where a `<br>` should be rendered.

**Review by danroth27 — CHANGES_REQUESTED** [review]
I think for the .NET 5 version of this doc we should simplify instead of trying to capture the deltas from the 3.1 docs. The delta is just a point in time thing, and we've published an announcement to capture it. Instead, what I'd like to see is the Blazor Server and WebAssembly tables collapsed into a single table since in .NET 5 they are the same, with the IE rows removed and no mention of Edge Legacy. Just a simple statement of what is supported in .NET 5.

**Comment by guardrex** [comment]
@danroth27 Ready for another look. :eyes:

File: aspnetcore/blazor/supported-platforms.md [hunk 2]
```diff
@@ -5,40 +5,58 @@ description: Learn about the supported platforms for ASP.NET Core Blazor.
...
+ms.date: 10/01/2020
 no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: blazor/supported-platforms
 ---
 # ASP.NET Core Blazor supported platforms
 
 By [Luke Latham](https://github.com/guardrex)
 
-## Browser requirements
+::: moniker range=">= aspnetcore-5.0"
```
Comment (danroth27) [inline aspnetcore/blazor/supported-platforms.md:17]:
```suggestion

Blazor Server and Blazor WebAssembly are supported in the following browsers:

```

**Review by danroth27 — APPROVED** [review]
One minor suggestion - other wise looks good! 👍

File: aspnetcore/blazor/supported-platforms.md (hunk 2 above)
Comment (guardrex) [reply aspnetcore/blazor/supported-platforms.md:17]: I'll make a minor edit. MS style is no colon before a table. They just want a period.

**Pull request merged**